addons/l10n_ar/views/report_invoice.xml

```diff
@@ -272,6 +272,30 @@
             <attribute name="t-call">l10n_ar.document_tax_totals</attribute>
         </t>
 
+        <div id="qrcode" position="after">
+            <!-- RG 5614/2024: Show ARCA VAT and Other National Internal Taxes -->
+            <t t-set="l10n_ar_custom_tax_summary" t-value="o._l10n_ar_get_invoice_custom_tax_summary_for_report()"/>
+            <div class="l10n_ar_tax_details col-sm-7 col-md-6" t-if="l10n_ar_tax_details">
```

Comment:
@zaoral I moved the `t-if` from `table` to this `div` to avoid an empty html block

Reply:
I made some test (directly in the PR) to fix the preview of the data we need, and found out that the table width of the tax detail is to short. not sure if a problem because in the right we are showing things that we do not need.  

![image](https://github.com/user-attachments/assets/5b403829-546c-4eb6-ac98-3207e6aa034c)

We are specting to be something like this (to the end, before the footer, after the odoo regular totals)
![image](https://github.com/user-attachments/assets/06862c5f-2839-41db-928a-44089603beb0)